packages/govuk-frontend-review/src/app.mjs

```diff
@@ -143,22 +143,23 @@ export default async () => {
     '/components/:componentName/:exampleName?/preview',
     function (req, res, next) {
       // Find the data for the specified example (or the default example)
-      const componentName = req.params.componentName
-      const exampleName = req.params.exampleName || 'default'
+      res.locals.componentName = req.params.componentName
+      res.locals.exampleName = req.params.exampleName || 'default'
```

Comment:
@owenatgov Not a blocker, but just a reminder for us to revisit this

So we're not assigning these in multiple places, I'll do a follow up PR to move parameter parsing into `app.param()` then other routes like `'/full-page-examples/:exampleName'` in **full-page-examples.mjs** pick it up

#### Handler for `app.param('componentName')`

```mjs
app.param('componentName', function (req, res, next, componentName) {
  const exampleName = 'default'

  // Find all fixtures for component
  const componentFixtures = componentsFixtures.find(
    ({ component }) => component === componentName
  )

  // Find default fixture for component
  const componentFixture = componentFixtures?.fixtures.find(
    ({ name }) => name === exampleName
  )

  // Add response locals
  res.locals.componentName = componentName
  res.locals.componentFixtures = componentFixtures
  res.locals.componentFixture = componentFixture
  res.locals.exampleName = 'default'

  next()
})
```

#### Handler for `app.param('exampleName')`

```mjs
app.param('exampleName', function (req, res, next, exampleName) {
  const { componentFixtures } = res.locals

  // Replace default fixture with named example
  const componentFixture = componentFixtures?.fixtures.find(
    ({ name }) => nunjucks.filters.slugify(name) === exampleName
  )

  // Update response locals
  res.locals.componentFixture = componentFixture
  res.locals.exampleName = exampleName

  next()
})
```